README.md

```diff
@@ -1,10 +1,9 @@
 ![GitHub contributors](https://img.shields.io/github/contributors/googlemaps/maps-sdk-for-ios-samples)
 ![Apache-2.0](https://img.shields.io/badge/license-Apache-blue)
 
-# Google Maps SDK for iOS and Google Places API for iOS sample code
+# Google Maps SDK for iOS, Google Places API for iOS, and Google Navigation API for iOS sample code
```

Comment:
Finally we update our outdated product name reference!

Please change to Google Places SDK for iOS and Google Navigation SDK for iOS

Reply:
Done